egs/librispeech/ASR/conv_emformer_transducer_stateless/test_emformer.py

```diff
@@ -0,0 +1,122 @@
+import torch
```

Comment:
```suggestion
#!/usr/bin/env python3

import torch
```

And please also add copyright info.